libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/route/line/api/VanishingRouteLineRoboTest.kt

```diff
@@ -176,10 +176,9 @@ class VanishingRouteLineRoboTest {
 
     @Test
     fun getTraveledRouteLineExpressions_withRestrictedLineExpressionData() {
-        val expectedRestrictedExpression = "[step, [line-progress], [rgba, 0.0, 0.0, 0.0, 0.0]," +
-            " 0.0, [rgba, 0.0, 0.0, 0.0, 0.0], 0.44865144220494346, [rgba, 0.0, 0.0, 12.0, 0.0]," +
-            " 0.468779750455607, [rgba, 0.0, 0.0, 0.0, 0.0], 0.5032854217424586, " +
-            "[rgba, 0.0, 0.0, 12.0, 0.0], 0.5207714038134984, [rgba, 0.0, 0.0, 0.0, 0.0]]"
+        val expectedRestrictedExpression = "[step, [line-progress], [rgba, 0.0, 0.0, 0.0, 0.0], " +
+            "0.0, [rgba, 0.0, 0.0, 0.0, 0.0], 0.4476941554901612, [rgba, 0.0, 0.0, 12.0, 0.0], " +
+            "0.5196445159361185, [rgba, 0.0, 0.0, 0.0, 0.0]]"
```

Comment:
This looks like a regression. The `route-with-restrictions.json` route has one intersection at `geometry_index = 17` with restriction, then followed by an intersection at index `geometry_index = 18` that doesn't have a restriction, and then again at index `19` we have a restriction. So all in all we should have 2 independent restricted sections (as the old test asserted) but we don't anymore.